.github/workflows/js-ci.yml

```diff
@@ -327,6 +327,13 @@ jobs:
           name: admin-ui-server-log-${{ matrix.container }}-${{ matrix.browser }}
           path: ~/server.log
 
+      - name: Upload Cypress videos
```

Comment:
Can we skip this on keycloak-private? probably also set a much shorter rentention days than the default (90)? 10 or so is probably sufficient isn't it?

Reply:
Added a condition so `keycloak-private` is excluded, as well as a retention of 10 days.